src/admin-views/aggregator/settings.php

```diff
@@ -142,6 +142,51 @@
 			'parent_option' => Tribe__Events__Main::OPTIONNAME,
 			'options' => $yes_no_options,
 		),
+		'tribe_aggregator_default_import_limit_type' => array(
+			'type' => 'dropdown',
+			'label' => esc_html__( 'Import limit type', 'the-events-calendar' ),
+			'tooltip' => esc_html__( 'Limit the number of imported events by number, date range or not at all; on slower websites this may impact the success of imports. Selecting a shorter time period or a smaller number  of events may improve results.', 'the-events-calendar' ) . ' ' . sprintf( '<a href="%1$s" target="_blank">%2$s</a>', esc_attr( 'https://theeventscalendar.com/knowledgebase/other-url-import-errors-in-event-aggregator' ), esc_html( 'Learn more.' ) ),
```

Comment:
Missing a comma and there's an extra space. How about:

```
Limit the number of imported events by number, date range, or not at all; on slower websites this may impact the success of imports. Selecting a shorter time period or a smaller number of events may improve results.
```

Reply:
Let's also use an m.tri.be link for the knowledge base article: http://m.tri.be/19rd

Reply:
Actually...I'm not sure if that KB article goes with it (as you alluded to in your comment on this PR). @barryhughes, do you think a KB article specifically targeted at limiting imported events by number/date range is warranted, or does the above tooltip text have enough info?